modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,351 @@
+
+As a cluster administrator, you can install the OpenNESS Operator for Intel® FPGA PAC N3000 by using the {product-title} CLI or the web console.
+
+[id="programmimg-n3000-with-a-vran-bitstream_{context}"]
+== Programming the N3000 with a vRAN bitstream
+
+As a cluster administrator, you can program the Intel® FPGA PAC N3000 with a vRAN 5G bitstream. This bitstream exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices used to accelerate the Forward Error Correction (FEC) in the vRAN workload.
+
+[NOTE]
+====
+The role of Forward Error Correction (FEC) is to correct transmission errors, where certain bits in a message can be lost or garbled. Messages can be lost or garbled due to noise in the transmission media, interference, or low signal strength.
+Without FEC, a garbled message would have to be resent, adding to the network load and impacting both throughput and latency.
+====
+
+.Prerequisites
+
+* Intel FPGA PAC N3000 card
+* Performance Addon Operator with RT kernel configuration
+* Node or nodes installed with the N3000 Operator
+* Log in as a user with `cluster-admin` privileges
++
+[NOTE]
+====
+All the commands run in the `vran-acceleration-operators` namespace.
+====
+
+.Procedure
+
+. Verify that the N3000 Operator is installed and that the pods are running:
++
+[source,terminal]
+----
+$ oc get csv
+----
++
+.Example output
+[source,terminal]
+----
+NAME            DISPLAY                                         VERSION REPLACES    PHASE
+n3000.v1.1.0    OpenNESS Operator for Intel® FPGA PAC N3000     1.1.0               Succeeded
+----
+
+. Verify that the pods are running:
++
+[source,terminal]
+----
+$ oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                        READY       STATUS          RESTARTS    AGE
+fpga-driver-daemonset-8xz4c                 1/1         Running         0           15d
+fpgainfo-exporter-vhvdq                     1/1         Running         1           15d
+N3000-controller-manager-b68475c76-gcc6v    2/2         Running         1           15d
+N3000-daemonset-5k55l                       1/1         Running         1           15d
+N3000-discovery-blmjl                       1/1         Running         1           15d
+N3000-discovery-lblh7                       1/1         Running         1           15d
+----
+The following section provides information on the installed pods:
+* `fpga-driver-daemonset` provides and loads the required Open Programmable Accelerator Engine (OPAE) drivers
+* `fpgainfo-exporter` provides N3000 telemetry data for Prometheus
+* `N3000-controller-manager` applies N3000Node CRs to the cluster and manages all the operand containers
+* `N3000-daemonset` is the main worker application
```

Comment:
"main worker application" is imprecise.  Is this the software that exposes the VFs?  If yes, then please state that instead.